ROOTFS/usr/bin/waggle-publish-metric

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+import argparse
+import time
+import base64
+import json
+from typing import NamedTuple, Any
+from pathlib import Path
+import re
+import sys
+from uuid import uuid4
+
+try:
+    from time import time_ns
```

Comment:
why do you have to do this?  i am just curious.  i am sure its needed. just wondering.

Reply:
I added a comment inline for this now. It turns out the time_ns func was added in Python 3.7. In order to adhere to the internal ns timestamp, I scale up time to ns units here.

This is something we may want to revisit in the future. I think the precision from this approximation is good enough for system metrics.

(The plugin images all run Python >= 3.7 in case you were wondering.)